x-pack/test/functional/config.js

```diff
@@ -138,6 +138,7 @@ export default async function ({ readConfigFile }) {
         '--server.uuid=5b2de169-2785-441b-ae8c-186a1936b17d',
         '--xpack.xpack_main.telemetry.enabled=false',
         '--xpack.security.encryptionKey="wuGNaIhoMpk5sO4UBxgr3NyW1sFcLgIf"', // server restarts should not invalidate active sessions
+        '--xpack.canvas.enabled=false'
```

Comment:
can we adda a todo with a link to the GitHub issue here?

Reply:
Yup, you got it (as long as I can merge without letting CI run again)

Reply:
haha, 👍